class-vip-support-user.php

```diff
@@ -436,8 +436,10 @@ public function action_password_reset( $user ) {
 			return;
 		}
 
-		$this->mark_user_email_verified( $user->ID, $user->user_email );
-		$this->promote_user_to_vip_support( $user->ID );
+		if ( is_proxied_automattician() ) {
+			$this->mark_user_email_verified( $user->ID, $user->user_email );
```

Comment:
I opted to remove the code which verifies the user email as part of the password reset step, as this functionality might have allowed non-VIP Automatticians to be accidentally assigned the VIP Support role.